CNAME

```diff
@@ -1 +0,0 @@
-thelounge.chat
```

Comment:
I just realized: would this break the redirect from thelounge.github.io to thelounge.chat?
If so I'll remove the commit.

Reply:
I guess it's not needed when using netlify?

Reply:
Removed. A year from now or something, we can remove this, and rename the repo (which still builds the GitHub Pages for now, that can't be disabled and if we remove we lose the redirect, _I think_). Nbd for now.